### ISSUE TITLE
Is this deprecated?

### ISSUE THREAD

**Issue by ghost** [plguhur/jm-django#5]
Is https://github.com/MieuxVoter/mvapi replacing this? There are many repositories and I don't know which one is still actual and which one to use.

**Comment by guhur** [comment]
Yes let's me archive it. Mvapi is right now online but we are actively working on a new specification (Mv-api-spec)
